docs/index.md

```diff
@@ -25,7 +25,7 @@
 * [Directories](./introduction/directories.md)
 * [Cron](./introduction/cron.md)
 * [Using Form Types](./introduction/using-form-types.md)
-* [Friendly URL](./introduction/friendly-url.md)
+* [Friendly URL, Breadcrumb navigation](introduction/friendly-url-front-end-breadcrumb-navigation.md)
```

Comment:
The label is not the same as the name of the article.

Reply:
Fixed